app/src/main/java/org/mozilla/focus/topsites/TopSites.kt

```diff
@@ -152,7 +151,7 @@ private fun TopSiteFaviconCard(topSite: TopSite) {
                     horizontalAlignment = Alignment.CenterHorizontally
                 ) {
                     Text(
-                        text = topSite.url.getRepresentativeCharacter(),
+                        text = if (topSite.title.isNullOrEmpty()) "" else topSite.title?.get(0).toString(),
```

Comment:
The title of a website can be empty but there should always be an URL.
Maybe it would be more helpful to keep the old functionality of getting a letter from the URL when the title is empty instead of showing a blank favicon?

Reply:
Done